codice/ddf
# DDF-04528 Migrating metacard-overwrite to React

#### What does this PR do?
Migrates the metacard-overwrite component to React
#### Who is reviewing it? 

@Bdthomson 
@andrewzimmer 
@hayleynorton 
@adimka 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@djblue

#### How should this be tested?
1. View an item in the inspector, and click on the overwrite tab
2. Overwrite the item and hit the back button to be able to overwrite again
2. Ensure that that there are no regressions in the component
#### Any background context you want to provide?
#### What are the relevant tickets?

For GH Issues:
Fixes: #4528 

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
